docs/guide/javaee/jms.adoc

```diff
@@ -22,18 +23,18 @@ Alternatively you could use a CLI script to add the queue.
 [source,options="nowrap"]
 $ jms-queue add --queue-address=WildFlyCamelQueue --entries=queue/WildFlyCamelQueue,java:/jms/queue/WildFlyCamelQueue
 
-Or, you could create a `messaging-deployment` configuration within a custom jms.xml deployment descriptor. See section 'Deployment of -jms.xml files' within the {wildfly} messaging subsystem documentation for more information.
+Also, you can create a `messaging-deployment` configuration within a custom jms.xml deployment descriptor. See section 'Deployment of -jms.xml files' within the {wildfly} messaging subsystem documentation for more information.
 
 #### Camel route configuration
-The following JMS producer and consumer examples make use of {wildfly}'s embedded ActiveMQ Artemis sever to publish and consume messages to and from destinations.
+The following JMS producer and consumer examples make use of {wildfly}'s embedded ActiveMQ Artemis server to publish and consume messages to and from destinations.
 
 The examples also use CDI in conjunction with the camel-cdi component. JMS ConnectionFactory instances are injected into the Camel RouteBuilder through JNDI lookups.
 
 ##### JMS Producer
 
-The RouteBuilder begins by injecting the `DefaultJMSConnectionFactory` connection factory from JNDI. If you're wondering where the connection factory is defined, look at the {wildfly} XML configuration and you'll see it is defined within the messaging subsystem.
+Start the RouteBuilder, by injecting the DefaultJMSConnectionFactory connection factory from JNDI. Under the JBoss EAP XML configuration, you can find the connection factory, within the messaging subsystem.
```

Comment:
'Start the RouteBuilder' is a little confusing because it implies the the instructions which follow some how start the RouteBuilder, which is not the case.

Maybe we should start the sentence with 'The DefaultJMSConnectionFactory connection factory is injected into the RouteBuilder from JNDI'.

Please also remove wording 'JBoss EAP' and replace with '{wildfly}'.

Reply:
fixed as suggested